### PR TITLE
feat(ios,android): expose Ti.UI.Slider „tintColor“ and „trackTintColor“

### PR DESCRIPTION
**Pull request by hansemannn**
**JIRA:**
https://jira.appcelerator.org/browse/TIMOB-27272

**Summary:**
Added new properties "tintColor" and "trackTintColor" to `Ti.UI.Slider`.

---
**Test 1 - Set Colors Upon Creation:**
1. Build and run the below on iOS.
2. Verify that the left side of the slider is red.
3. Verify that the right side of the slider is green.
4. Repeat test on Android.

```javascript
var window = Ti.UI.createWindow({
	backgroundColor: "white",
});
window.add(Ti.UI.createSlider({
	min: 0.0,
	max: 1.0,
	value: 0.5,
	tintColor: "red",
	trackTintColor: "green",
}));
window.open();
```

---
**Test 2 - Set Colors After Creation:**
1. Build and run the below code on iOS.
2. Tap on the "Change Slider Color" button.
3. Verify left side of the slider bar changed to red.
4. Verify right side of the slider bar changed to green.
5. Repeat test on Android.

```javascript
var window = Ti.UI.createWindow({
	backgroundColor: "#CCCCCC",
});
var slider = Ti.UI.createSlider({
	min: 0.0,
	max: 1.0,
	value: 0.5,
});
window.add(slider);
var button = Ti.UI.createButton({
	title: "Change Slider Color",
	bottom: "30dp",
});
button.addEventListener("click", function() {
	slider.tintColor = "red";
	slider.trackTintColor = "green";
});
window.add(button);
window.open();
```

### CODE REVIEW

File: android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUISlider.java [hunk 1]
```diff
@@ -101,6 +102,12 @@ public void processProperties(KrollDict d)
 		if (d.containsKey("leftTrackImage") || d.containsKey("rightTrackImage")) {
 			updateTrackingImages(seekBar, d);
 		}
+		if (d.containsKey(TiC.PROPERTY_TINT_COLOR)) {
+			handleSetTintColor(TiConvert.toColor(d, TiC.PROPERTY_TINT_COLOR));
+		}
+		if (d.containsKey(TiC.PROPERTY_TINT_COLOR)) {
```
Comment (janvennemann) [inline android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUISlider.java]: I guess this should be

```suggestion
		if (d.containsKey(TiC.PROPERTY_TRACK_TINT_COLOR)) {
```

**Review by janvennemann — CHANGES_REQUESTED** [review]

File: android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUISlider.java (hunk 1 above)
Comment (hansemannn) [reply android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUISlider.java]: haha, yeah that was actually copy pasted from the TiUIProgressBar.java where it was probably also copy pasted from the tint color. I'll fix both together

**Comment by hansemannn** [comment]
@janvennemann Updated

**Review by janvennemann — APPROVED** [review]
LGTM!

@jquick-axway Can you please take a quick look at the Android changes too? I think they are fine, just so someone from the Android team approves this too. Thanks!

**Review by sgtcoolguy — APPROVED** [review]
Android changes look fine

**Comment by jquick-axway** [comment]
Sorry about the late response.

I see one parity issue. On Android, you can only set the property upon creation. On iOS, it looks like the color can be changed dynamically. On Android, we would need to add `tintColor` and `trackTintColor` property handlers in the `TiUISlider.propertyChanged()` method. This may be useful if we want to support changing dark/light themes dynamically in the future.

**Comment by jquick-axway** [comment]
@hansemannn, are you willing to make the change I suggested above? This is in regards to it being creation-only properties on Android versus iOS supports changing them dynamically. This is the only thing that's blocking this PR. I can get someone else to make the final changes to this PR if you don't have time. Thanks.

**Comment by hansemannn** [comment]
Please assign it internally, thank you!

**Comment by jquick-axway** [comment]
Updated PR:
- Modified Android tint properties to be set dynamically like iOS.
- Added unit test.

**Review by jquick-axway — APPROVED** [review]
CR: Pass
FR: Pass

**Comment by ssaddique** [comment]
FR: Passed.

Studio Ver:     5.1.4.201909061933
SDK Ver:        8.3.0 (from PR)
Mac OS Ver:     10.14.6
Xcode Ver:      Xcode 11.0
Appc NPM:       4.2.15
Appc CLI:       7.1.1
Node Ver:       10.16.3
NPM Ver:        6.11.3
Emulators:       ⇨ Pixel 3 (API 28), iPhone 11 Pro (13.0)

**Pull request merged**